### DISCUSSION TITLE
How to set load output state?

### DISCUSSION THREAD

**Discussion by 19siro80** [softwarecrash/Victron2MQTT#25] [Q&A]
Hello there
 
I got victron2mqqt running on 1st attempt to access data from a smartsolar 20/100 in homeassistant. To take full advantage of this, I'd like to switch on and off the load output from the smarsolar device. Is victron2mqtt capable of sending a command? What is the correct mqtt command? Any help is appreciated as I couldn't find the information.

Thanks in advance.
siro

**Comment by softwarecrash** [comment]
take a look at the wiki
https://github.com/softwarecrash/Victron2MQTT/wiki/Remote-Control

**Reply by 19siro80** [reply]
Excellent. Thank you alot for your help.
I tried this and it partially worked.

<topic>/Remote_Control set to true or false work well if the RX port function in the Victron app is set to "remote control on/off". So I can successfully chance to state of the device over MQTT.

Using the same command if the RX port function is set to "load output on/off normal" it did not change anything. The load output does not change its state. 

What do I oversee?

**Reply by 19siro80** [reply]
Resolve!
In Victron app the load output setting must be set to always on. Only with this setting you can successfully switch the output through MQTT <topic>/Remote_Control set to true or false

Excellent solution. Now I can work on my automation in HA :)

**Reply by softwarecrash** [reply]
the settings deepens on your device what it can do
